src/pkg/bundle/deploy.go

```diff
@@ -512,15 +425,32 @@ func setTemplatedVariables(templatedVariables string, pkgVars map[string]string)
 }
 
 // addFileValue adds a key=filepath string to helm FileValues
-func addFileValue(helmFileVals []string, filePath string, override types.BundleChartVariable) ([]string, error) {
-	verifiedPath, err := formFilePath(override.Source, filePath)
+func (b *Bundle) addFileValue(helmFileVals []string, filePath string, override types.BundleChartVariable) ([]string, error) {
+	verifiedPath, err := formFilePath(getSourcePath(override.Source, b), filePath)
 	if err != nil {
 		return nil, err
 	}
 	helmVal := fmt.Sprintf("%s=%v", override.Path, verifiedPath)
 	return append(helmFileVals, helmVal), nil
 }
 
+// getSourcePath returns the path from where a value is set
+func getSourcePath(pathType valuesources.Source, b *Bundle) string {
```

Comment:
any reason to not make this a receiver function as well? (as opposed to passing in `b`)